payjoin/src/core/receive/mod.rs

```diff
@@ -312,7 +312,7 @@ impl PsbtContext {
     /// Finalization consists of two steps:
     ///   1. Remove all sender signatures which were received with the original PSBT as these signatures are now invalid.
     ///   2. Sign and finalize the resulting PSBT using the passed `wallet_process_psbt` signing function.
-    pub fn finalize_proposal(
+    pub(crate) fn finalize_proposal(
```

Comment:
as stated above this can be made private later on when some more shuffling is done.